enarx-keep-sgx-shim/src/elf.rs

```diff
@@ -0,0 +1,33 @@
+// SPDX-License-Identifier: Apache-2.0
+
+pub const MAGIC: [u8; 4] = [0x7f, 0x45, 0x4c, 0x46];
```

Comment:
Curious about the `MAGIC`.

Reply:
https://en.wikipedia.org/wiki/Executable_and_Linkable_Format#File_header

Also: https://en.wikipedia.org/wiki/Magic_number_(programming)

The `MAGIC` number lets a program like `file` scan the first four bytes of a file and detect that it is an ELF binary. In our case we're using them as a sanity test to make sure we're looking at the right memory.